src/plugins/PluginAPI.cpp

```diff
@@ -301,7 +301,12 @@ APICALL std::vector<SFunctionMatch> HyprlandAPI::findFunctionsByName(HANDLE hand
             count++;
         }
 
-        return SYMBOLSDEMANGLED.substr(pos, SYMBOLSDEMANGLED.find('\n', pos + 1) - pos);
+        // Skip the newline char itself
+        if (pos != 0) {
```

Comment:
my man no {} around short ifs

Reply:
Oh yeah, forgot you don't need those in C++, still not fully used to all the lang's conventions. Fixed.